test/lwsp/Makefile

```diff
@@ -0,0 +1,65 @@
+# Python configuration file
+# REVCFG ?= ./rev-memh.py
+REVCFG ?= ../rev-model-options-config.py
+
+# Rev toolchain
+CC=riscv64-unknown-elf-g++
```

Comment:
This should be `CXX=${RVCXX}` once my [PR](https://github.com/tactcomplabs/rev/pull/291) is merged.

`CC` should only be used for C compilers; `CXX` for C++ compilers.

You may not need to create your own `Makefile` -- `test/CMakeLists.txt` has a general way of handling tests.

Reply:
It is already in my [PR](https://github.com/tactcomplabs/rev/pull/291)